src/main/java/core/basesyntax/FileWork.java

```diff
@@ -9,6 +15,24 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        File file = new File(fileName);
+        ArrayList<String> textFromFile = new ArrayList<>();
+        final char StartsWith = 'w';
```

Comment:
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html
Check how we should create constants

Reply:
?